lab_1/quality-check.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -
```

Comment:
this script assumes that you have already prepared env specifically for its run: correct node version used,  required tools installed, node_modules in place and so on.

it has a rather low value without several steps done beforehand